packages/perseus/src/util/parse-perseus-json/perseus-parsers/perseus-item.ts

```diff
@@ -6,23 +6,47 @@ import {
     enumeration,
     number,
     object,
+    pipeParsers,
     record,
 } from "../general-purpose-parsers";
 
 import {parseHint} from "./hint";
 import {parsePerseusRenderer} from "./perseus-renderer";
 
 import type {PerseusItem} from "../../../perseus-types";
-import type {Parser} from "../parser-types";
+import type {ParseContext, Parser, ParseResult} from "../parser-types";
 
 export const parsePerseusItem: Parser<PerseusItem> = object({
     question: parsePerseusRenderer,
     hints: array(parseHint),
-    answerArea: record(enumeration(...ItemExtras), boolean),
+    answerArea: pipeParsers(object({}))
+        .then(migrateAnswerArea)
+        .then(record(enumeration(...ItemExtras), boolean)).parser,
```

Comment:
Just so I understand, without `migrateAnswerArea` the `enumeration()` parser tries to parse `type: string` and/or `options: object` and fails parsing?

Reply:
Correct. `type` and `options` aren't listed in `ItemExtras`, so the enumeration parser will not match them.